.changeset/wise-chicken-divide.md

```diff
@@ -0,0 +1,9 @@
+---
+'@talend/react-datagrid': major
+'@talend/react-components': patch
```

Comment:
one changeset per changelog. changeset is not able to split the comments below.
This is ok if you want the same changelog in every packages whcih is not the case here.